### DISCUSSION TITLE
Apply dark themes to breakout rooms, captions/subtitles dialog, etc.

### DISCUSSION THREAD

**Discussion by cool0707** [igniterealtime/openfire-pade-plugin#298] [General]
I'd like to change `tingle.css` to make the dialogs added with `custom_ofmeet.js` such as breakout rooms to be dark themed like the Jitsi Meet dialogs, how about that?

**Comment by deleolajide** [comment]
Yes please 😊

**Comment by gjaekel** [comment]
In lack of a test community, I wasn't able to check the Breakout Room panel. But from the Contacts Manager I know for sure that it still don't fits to the available space: One have to scroll to reach the control buttons.

![image](https://user-images.githubusercontent.com/19855721/113108430-45e41780-9205-11eb-89ae-cee2b13bcc6c.png)

Moving the buttons to the top isn't common UI style, I think. But on the top, there might be a "live" filter slot, which may hide other but the one with contain the search term.
